packages/flutter/lib/src/animation/curves.dart

```diff
@@ -112,7 +112,7 @@ class SawTooth extends Curve {
 }
 
 /// A curve that is 0.0 until [begin], then curved (according to [curve] from
-/// 0.0 to 1.0 at [end], then 1.0.
+/// 0.0 to 1.0) until [end], then 1.0.
```

Comment:
i've no idea what the original intent of this was, but i don't think the proposed text makes it any clearer (though it does fix the dangling parenthesis).

Maybe:

```
A curve that is 0.0 until [begin], then curved (according to [curve]) from 0.0 at [begin] to 1.0 at [end], then remains 1.0 past [end].
```
...?